parsl/dataflow/dflow.py

```diff
@@ -262,6 +262,12 @@ def _create_task_log_info(self, task_record):
                 stderr_name = ""
         else:
             stderr_name = ""
+            # TODO:
+            # When stdout_spec is a File, the above fails, but we can't then use str, because str for a File
+            # gives the localpath, which doesn't exist on the submit side for a non-file: File object.
+            # Reporting the stderr path here probably should be done differnetly for File objects, perhaps
+            # reporting the URL without using get_std_fname_mode which only makes sense in the case of
+            # shared filesystem stdout/stderrs?
```

Comment:
Typically, when I see a TODO in a code base, it's a clue to me that we need a ticket.  That is, recognizing that, when I'm planning my next foray between tasks, I don't (usually) take my cues of "what to do next"  by searching for TODO comments, the best place for task visibility is a well-formed ticket.  Especially if planning with a team.

Parsl is not developed via a more formal sprintly arrangement, so this is perhaps "just fine," but I observe it nonetheless.

Reply:
oh i thought i had fixed this before I pushed it up -- that's unintentional...

Reply:
should be addressed now, with testing building on the tests introduced in #3363